scripts/dataset_processing/tts/sfbilingual/get_data.py

```diff
@@ -38,7 +38,7 @@ def get_args():
     parser.add_argument(
         "--manifests-path", type=Path, help="where the resulting manifests files will reside", default="./"
     )
-    parser.add_argument("--val-size", default=0.005, type=float, help="eval set split")
+    parser.add_argument("--val-size", default=0.01, type=float, help="eval set split")
```

Comment:
Why change this if the tutorial suggests using the original split percentages?

Reply:
actually, all our models trained previously override this as val/test == 0.01. Fixed here because it is a minor change. I will revise the ones in the tutorial.